events/syscall/disallowed_ssh_connection_non_standard_port.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(DisallowedSSHConnectionNonStandardPort)
+
+func DisallowedSSHConnectionNonStandardPort(h events.Helper) error {
+    // non_standard_port : 443
+    cmd := exec.Command("timeout", "1s", "ssh", "user@example.com", "-p", "443")
```

Comment:
Check if `ssh` is present?

Reply:
@FedeDP I made changes and rule triggering successfully and throwing error too when ssh not present.